sopc_components_altera.xml

```diff
@@ -1,4 +1,16 @@
 <SOPC2DTS_COMPONENTLIB vendor="ALTR">
+	<S2DComponent classname="mm_ccb"
+		group="bridge">
+	</S2DComponent>
+	<S2DComponent classname="altera_i2cslave_to_avlmm_bridge"
+		group="bridge">
+	</S2DComponent>
```

Comment:
Is the i2cslave really a bridge from a devicetree pov?
I think this should be an end-node as I _think_ we wouldn't want to describe what is going on at the other side of that bridge...

Reply:
Sure, could you suggest how to define it in the XML?  
Should I specify a different group? Which? Or should it be simply "ignore"?

Reply:
You _could_ use ignore, but there's probably a use case for some external system which drives the i2c master to describe the stuff behind this bridge. So just pick something that's not bridge :) i2c2avalonmaster (?)
Bridge is a special case were sopc2dts tries to look past the other end of a bridge.

Fun fact: I initially created sopc2dts for this specific case: I had a device-tree enabled PPC processor that connected over PCIe to a StratixIV and the loaded the FPGA devicetree from there...

Reply:
Ok - changed group to `i2c2avalonmaster`.  
What I was missing was that only bridge has "special meaning" in sopc2dts, and I'm free to select any other group name for anything that is not a bridge.

Reply:
There are some other groups that are semi-magical but bridge is the "worst"